relayer/relays/parachain/query_events.go

```diff
@@ -73,6 +73,10 @@ func (q *QueryClient) QueryEvents(ctx context.Context, api string, blockHash typ
 		return nil, err
 	}
 
+	log.WithFields(log.Fields{
+		"inputItems": items,
+	}).Info("parachain.QueryEvents")
```

Comment:
Maybe log this out as debug information.
```suggestion
	}).Debug("parachain.QueryEvents")
```